index.bs

```diff
@@ -41,6 +57,7 @@ spec:dom-ls
 spec: css-values-3; type: dfn
 	text: absolute length
 	text: dimension
+spec: embedded-content; type: element; text: iframe
```

Comment:
There's no spec called "embedded-content".  Bikeshed only knows of one "iframe" reference, so you shouldn't need this line at all.  (If you didn't know, there's a markup shorthand for element refs - `<{iframe}>`.)